wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -417,19 +417,76 @@ footer,
             display: none;
         }
 
-        img {
-            width: 60px;
+        .wagtail-logo-container {
             float: none;
             margin: auto;
             display: block;
+            width: 60px;
+            height: 75px;
+            position: relative;
+            transition: all 0.25s cubic-bezier(0.28, 0.15, 0, 2.1);
+
+            img {
+                left: 0;
+                top: 0;
+                width: 60px;
+                position: absolute;
+                transition: all 0.25s cubic-bezier(0.28, 0.15, 0, 2.1);
+            }
+
+            img.wagtail-logo {
+                display: none; // This is the static logo (mobile only); use the animated one on desktop
+             }
+
+            img.wagtail-logo-eye {
+                display: inline;
+            }
+
+            img.wagtail-logo-eyeclosed {
+                display: none;
+            }
         }
 
-        img.wagtail-logo {
-            transition: all 0.25s cubic-bezier(0.28, 0.15, 0, 2.1);
+
+        // Wagtail animation:
+        @keyframes nod {
+            0% {
+                transform: rotate(2deg);
+            }
+
+            100% {
+                transform: rotate(6deg);
+            }
         }
 
-        img.wagtail-logo:hover {
-            transform: rotate(4deg);
+        .wagtail-logo-container:hover {
+            // 'Nod' animation
```

Comment:
This and a few of the other comments below don't seem necessary – maybe just rename the animations if in doubt?